gramps/gui/editors/editfamily.py

```diff
@@ -793,7 +793,11 @@ def _create_tabbed_pages(self):
                                             self.uistate,
                                             self.track,
                                             self.obj.get_lds_ord_list())
-        self._add_tab(notebook, self.lds_embed)
+
+        if config.get('interface.use-lds') == False and self.lds_embed.is_empty() == True:
```

Comment:
You can simplify this to:
`if not config.get('interface.use-lds') and self.lds_embed.is_empty():`